pkgs/development/tools/misc/blisp/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  IOKit,
+  testers,
+  blisp,
+}:
+let
+  version = "0.0.3";
+in stdenv.mkDerivation {
+  pname = "blisp";
+  inherit version;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "blisp";
  version = "0.0.3";
```

the rec keyword makes this a recursive function and ever attribute set like pname and version available in the same scope.